src/Features/CSharp/Portable/Completion/CompletionProviders/ObjectCreationCompletionProvider.cs

```diff
@@ -56,20 +56,18 @@ protected override SyntaxNode GetObjectCreationNewExpression(SyntaxTree tree, in
             return CSharpSyntaxContext.CreateContext(document.Project.Solution.Workspace, semanticModel, position, cancellationToken);
         }
 
-        protected override async Task<ImmutableArray<ISymbol>> GetPreselectedSymbolsWorker(SyntaxContext context, int position, OptionSet options, CancellationToken cancellationToken)
+        protected override async Task<ImmutableArray<(ISymbol, CompletionItemRules)>> GetPreselectedSymbolsWorker(SyntaxContext context, int position, OptionSet options, CancellationToken cancellationToken)
         {
             var result = await base.GetPreselectedSymbolsWorker(context, position, options, cancellationToken).ConfigureAwait(false);
-            if (result.Any())
+            var builder = ArrayBuilder<(ISymbol, CompletionItemRules)>.GetInstance();
+
+            foreach (var (symbol, _) in result)
             {
-                var type = (ITypeSymbol)result.Single();
-                var alias = await type.FindApplicableAlias(position, context.SemanticModel, cancellationToken).ConfigureAwait(false);
-                if (alias != null)
-                {
-                    return ImmutableArray.Create(alias);
-                }
+                var alias = await symbol.FindApplicableAliasAsync(position, context.SemanticModel, cancellationToken).ConfigureAwait(false);
+                builder.Add((alias ?? symbol, CompletionItemRules.Default));
```

Comment:
seems odd that you're overriding the completion rules set by the base type.  Is that intentional?  If so, can you give a comment explaning why?